lib/sqlitex/row.ex

```diff
@@ -22,10 +22,21 @@ defmodule Sqlitex.Row do
     nil
   end
 
+  # date is empty ""
+  defp translate_value({date, "date"}) when date=="", do: to_date(date)
```

Comment:
You're still calling `to_date` with the empty string here, so this isn't much different from the original code.  You should maybe just return nil?